test/functional/editions_controller_test.rb

```diff
@@ -39,6 +39,53 @@ class EditionsControllerTest < ActionController::TestCase
     end
   end
 
+  context "#request_amendments" do
```

Comment:
Context needs updating now that we've renamed the routes and controller methods.
```suggestion
  context "#request_amendments_page" do
```